entgql/template/where_input.tmpl

```diff
@@ -20,7 +20,6 @@ in the LICENSE file in the root directory of this source tree.
 import (
     "{{ $.Config.Package }}/predicate"
 	{{- range $n := $gqlNodes }}
-        {{ template "import/types" $n }}
```

Comment:
This template imports any packages required by custom `GoType`s. See https://github.com/ent/ent/blob/master/entc/gen/template/import.tmpl#L45-L57. 

Maybe change it like this?
```diff
-         {{ template "import/types" $n }}
+         {{- template "import/types" $n }}
```